src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -2102,11 +2103,26 @@ public static boolean getCoerceReadConsistencyAll()
         return conf.coerce_read_consistency_all;
     }
 
+    public static Set<String> getCoerceReadConsistencyAllKeyspaces()
+    {
+        return conf.coerce_read_consistency_all_keyspaces;
+    }
```

Comment:
Nit: to simplify consuming logic, we could have this function return all keyspaces if the set is empty. Not strongly opinionated though considering this is just for a rc, so feel free to leave as is if you prefer